File: 프로그래머스/0/181834. l로 만들기/l로 만들기.java
class Solution {
    public String solution(String myString) {
        String answer = "";

        // a = 97 ... l = 108
        for(int i = 0; i < myString.length(); i++) {
            answer += myString.charAt(i) < 108 ? "l" : myString.charAt(i);
        }
        
        return answer;
    }
}

// 다른 풀이
return myString.replaceAll("[^l-z]", "l");

// 다른 풀이
return myString.chars()
               .mapToObj(i -> Character.toString(Integer.max(i, 'l')))
               .collect(Collectors.joining());

File: 프로그래머스/0/181834. l로 만들기/l로 만들기.py
def solution(myString):
    answer = ''
    
    for str in myString:
        if(str < 'l'):
            answer += 'l'
        else:
            answer += str
    
    return answer

# 다른 풀이
return myString.translate(str.maketrans('abcdefghijk', 'lllllllllll'))
# str.replace(old, new) : 모든 old 문자열을 new 문자열로 치환

# 다른 풀이
answer = [x if x > 'l' else 'l' for x in myString]
return ''.join(answer)
